### PR TITLE
Rocminfodocs reorg

### PR DESCRIPTION
**Pull request by Rmalavally**

### CODE REVIEW

**Review by dayatsin-amd — APPROVED** [review]

File: docs/index.rst [hunk 1]
```diff
@@ -15,7 +15,7 @@ For more details, see `GitHub. <https://github.com/ROCm/rocminfo>`_
 
   .. grid-item-card:: Install
```
Comment (dayatsin-amd) [inline docs/index.rst:16]: .. grid-item-card:: Install

Should this line be:
.. grid-item-card:: Build

**Review by dayatsin-amd — APPROVED** [review]

**Review by dayatsin-amd — APPROVED** [review]

**Review by dayatsin-amd — APPROVED** [review]

**Review by dayatsin-amd — APPROVED** [review]

**Review by dayatsin-amd — APPROVED** [review]

**Review by dayatsin-amd — APPROVED** [review]

**Review by dayatsin-amd — APPROVED** [review]

**Review by dayatsin-amd — APPROVED** [review]

File: docs/index.rst (hunk 1 above)
Comment (Rmalavally) [reply docs/index.rst]: Fixed. Thanks, David.

File: docs/how-to/use-rocm-agent-enumerator.rst [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+  :description: Install ROCmInfo
+  :keywords: install, rocminfo, AMD, ROCm
+
+
+Using ROCm agent enumerator
+-----------------------------
+
+You can use ROCmInfo with the following commands:
+
+.. code-block::
```
[the remark this answers is not in the document]
Comment (Rmalavally) [reply docs/how-to/use-rocm-agent-enumerator.rst:11]: I shared the web link with you. Please review and let me know. Thanks so much.

**Review by lpaoletti — APPROVED** [review]
This looks good to go.

**Comment by lpaoletti** [comment]
Actually, we do need one change. 
Looks like it's officially named ROCm Info - no space. 

image

File: docs/how-to/use-rocm-agent-enumerator.rst [hunk 3]
```diff
@@ -0,0 +1,281 @@
+.. meta::
+  :description: Install ROCmInfo
+  :keywords: install, rocminfo, AMD, ROCm
+
+
+Using ROCm agent enumerator
+-----------------------------
+
+The following products support ROCm Info:
```
Comment (lpaoletti) [inline docs/how-to/use-rocm-agent-enumerator.rst:9]:
```suggestion
The following products support ROCmInfo:
```

File: docs/how-to/use-rocm-agent-enumerator.rst [hunk 4]
```diff
@@ -0,0 +1,281 @@
+.. meta::
+  :description: Install ROCmInfo
+  :keywords: install, rocminfo, AMD, ROCm
+
+
+Using ROCm agent enumerator
```
Comment (lpaoletti) [inline docs/how-to/use-rocm-agent-enumerator.rst:6]: Find this confusing. 
The title is about the agent enumerator - don't understand the relationship to ROCmInfo.

File: docs/install/build.rst [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    
+    cmake -DCMAKE_PREFIX_PATH=/opt/rocm ..
+    
+    make
+    
+    cd ..
+
+Upon a successful build, the binary, rocminfo, and the python script, rocm_agent_enumerator, will be in the `build` folder.
+
+ROCmInfo execution
```
Comment (lpaoletti) [inline docs/install/build.rst:34]: What does this section have to do with Installation?
This info sounds like what the tool actually returns.

File: docs/install/build.rst [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+-------------------
+
+"rocminfo" gives information about the HSA system attributes and agents.
```
Comment (lpaoletti) [inline docs/install/build.rst:37]: Why lowercase for rocminfo?
This description is more robust than the description on the ROCmInfo landing page. Might want to improve.

File: docs/index.rst [hunk 7]
```diff
@@ -1,6 +1,31 @@
...
+  :description: Install ROCmInfo
+  :keywords: install, rocminfo, AMD, ROCm
+
+ROCmInfo documentation
+*************************
+
+ROCmInfo is a ROCm application for reporting system information.
+
+
+For more details, see `GitHub. <https://github.com/ROCm/rocminfo>`_
```
Comment (lpaoletti) [inline docs/index.rst:11]: Should this be:
You can access ROCmInfo code on ...

File: docs/how-to/use-rocm-agent-enumerator.rst [hunk 8]
```diff
@@ -0,0 +1,281 @@
...
+                           filename where to find the "target.lst" file. This can be
+                           used in an install environment with sandbox, where
+                           execution of "rocminfo" is not possible.
+        2. target.lst : user-supplied text file. This is used in a container setting
+                      where ROCm stack may usually not available.
+        3. HSA topology : gathers the information from the HSA node topology in
+                          /sys/class/kfd/kfd/topology/nodes/
+        4. lspci : enumerate PCI bus and locate supported devices from a hard-coded
+                   lookup table.
+        5. rocminfo : a tool shipped with this script to enumerate GPU agents
```
Comment (lpaoletti) [inline docs/how-to/use-rocm-agent-enumerator.rst:259]: This is confusing
is "rocminfo" different than ROCmInfo?

File: docs/how-to/use-rocm-agent-enumerator.rst (hunk 3 above)
Comment (Rmalavally) [reply docs/how-to/use-rocm-agent-enumerator.rst]: Fixed. Thanks, @lpaoletti

File: docs/install/build.rst (hunk 6 above)
Comment (Rmalavally) [reply docs/install/build.rst:37]: Fixed. Thanks, @lpaoletti.

File: docs/how-to/use-rocm-agent-enumerator.rst (hunk 4 above)
Comment (Rmalavally) [reply docs/how-to/use-rocm-agent-enumerator.rst:6]: Added a description. Fixed.

File: docs/install/build.rst (hunk 5 above)
Comment (Rmalavally) [reply docs/install/build.rst:34]: Good point. Let me check with the Component team. @lpaoletti

File: docs/how-to/use-rocm-agent-enumerator.rst [hunk 9]
```diff
@@ -0,0 +1,281 @@
+.. meta::
+  :description: Install ROCmInfo
+  :keywords: install, rocminfo, AMD, ROCm
+
+
+Using ROCm agent enumerator
+-----------------------------
+
+The following products support ROCm Info:
+
```
[the remark this answers is not in the document]
Comment (Rmalavally) [reply docs/how-to/use-rocm-agent-enumerator.rst:12]: I did ask the Component team this question, and they said this task is supported only on these products, so the list of supported hardware is contextual.

File: docs/how-to/use-rocm-agent-enumerator.rst (hunk 8 above)
Comment (Rmalavally) [reply docs/how-to/use-rocm-agent-enumerator.rst:261]: Fixed.

**Pull request merged**